templates/invitation.html

```diff
@@ -1,5 +1,15 @@
-<h2>{{name}}</h2>
+<p>Hello {{name}}</p>,
```

Comment:
The comma probably doesn't want to be outside the `<p>`

Reply:
Damn you, Mr Semantic.